### PR TITLE
[13.0][MIG] sale_order_ubl

### PR DESCRIPTION
**Pull request by astirpe**
Depends on:

- [x] https://github.com/OCA/edi/pull/154

### CODE REVIEW

**Review by jgrandguillaume — APPROVED** [review]
LGMT thanks

**Review by sebalix — APPROVED** [review]
LG

File: sale_order_ubl/tests/test_ubl_generate.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def test_ubl_generate(self):
                 res = buo.get_xml_files_from_pdf(pdf_file)
                 if order.state in quotation_states:
                     filename = order.get_ubl_filename(
-                        'quotation', version=version)
+                        doc_type='quotation', version=version)
```
Comment (alexis-via) [inline sale_order_ubl/tests/test_ubl_generate.py]: doc_type is not a named arg in get_ubl_filename()

Comment (astirpe) [reply sale_order_ubl/tests/test_ubl_generate.py]: Done. Thanks!

**Pull request merged**